test/testinput/bufr_ncep_adpsfc.yaml

```diff
@@ -0,0 +1,542 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+# #
+# # This software is licensed under the terms of the Apache Licence Version 2.0
+# # which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "testinput/gdas.t12z.adpsfc_thinned.tm00.bufr_d"
+
+      exports:
+        variables:
+          timestamp:
+            datetime:
+              year: "*/YEAR[1]"
+              month: "*/MNTH[1]"
+              day: "*/DAYS[1]"
+              hour: "*/HOUR[1]"
+              minute: "*/MINU[1]"
+          durationOfTimeRelatingToDepthOfFreshSnow:
+            query: "*/.DTHDOFS"
+          durationOfTimeRelatingToMinimumTemperature:
+            query: "*/.DTHMITM"
+          durationOfTimeRelatingToMaximumTemperature:
+            query: "*/.DTHMXTM"
+          durationOfTimeRelatingToTotalPrecipitation:
+            query: "*/.DTHTOPC"
+          relationshipToHorizontalVisibility:
+            query: "*/.REHOVI"
+          PressureChangePast24Hour:
+            query: "*/24PC"
+          PressureChangePast3Hour:
+            query: "*/3HPC"
+          altimeterSettingQnh:
+            query: "*/ALSE"
+          characteristicOfPressureTendency:
+            query: "*/CHPT"
+          cloudAmount:
+            query: "*/CLAM"
+          latitude:
+            query: "[*/CLAT, */CLATH]"
+          longitude:
+            query: "[*/CLON, */CLONH]"
+          cloudType:
+            query: "[*/CLDSQ1/CLTP, */MTRCLD/CLTP, */CLTP[1], */BSYSCLD/CLTP, */BSYBCLD/CLTP]"
```

Comment:
I don't think you necessarily need to change anything, but I just wanted to note that BSYSCLD and BSYBCLD tend to be present together in the same subsets so that you will always get */BSYSCLD/CLTP. I just wanted to make sure this was intended and OK.

Reply:
@nicholasesposito @rmclaren pointed out something you need to pay attention to.  The CLTP data in BSYSCLD and BSYBCLD are cloud types for cloud layer data and cloud base data, respectively.  But they have the same mnemonic (CLTP) and in the same subset.  If you want to keep them in the YAML, you will need to query them separately, and give slight different variable names.  

The other option is to remove them from the YAML.  We want to keep the data used in GSI for this exercise.  Also, them these variables in the output IODA.  They may be all missing.

Reply:
I'll take them out for now while I figure out how best to split them up. NC000100, 101 and 102 have 5 different CLTP variables.